scripts/init.sh

```diff
@@ -42,13 +42,18 @@ for b in $PLUGIN_BINS; do
 done
 
 # Configure rp_filter
+echo "Configure default rp_filter loose... "
```

Comment:
Below set of changes will set `rp_filter` to `2` on all the interfaces and that might be a concern because we currently allow Cx to specify via tags the ENIs they don't want VPC CNI to manage. Same is true for P4d instances that support multiple network cards. Although, we only manage interfaces belonging to network card `0` this change will modify the settings on interfaces tied to non-zero network cards as well.

Reply:
Are there any extra information that we can use to reliably ignore the interfaces, for example via IMDS? Since we are setting the default rp_filter, any new interfaces created after this point gets the permissive rp_filter configuration.